src/Exception/ParameterNotAllowedException.php

```
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\PageBundle\Exception;
+
+class ParameterNotAllowedException extends SonataPageException
```

We don't need this

---

Why not? how do you group the exceptions? fro exaple

if I want to catch all exceptions from SonataPage like

```php
cache(SonataPageException $exception) ...
```

---

It's just that if we don't need https://github.com/sonata-project/SonataPageBundle/pull/1434#discussion_r919977982 we then don't need any custom exception.

---

Ok I got it :)